cmd/cgr-loader/cgr-loader.go

```diff
@@ -65,6 +65,10 @@ var (
 		"The delay before executing the commands if the redis cluster is in the CLUSTERDOWN state")
 	dbQueryTimeout = cgrLoaderFlags.String("query_timeout", utils.IfaceAsString(dfltCfg.DataDbCfg().Opts[utils.QueryTimeoutCfg]),
 		"The timeout for queries")
+	dbRedisTls               = cgrLoaderFlags.Bool(utils.RedisTLS, false, "Use a tls connection when connecting to redis")
```

Comment:
Please use as help message something like: "Enable TLS when connecting to Redis"